_posts/2018-04-11-making-a-cockpit-application.md

```diff
@@ -0,0 +1,60 @@
+---
+title: How to turn your Cockpit page into an Application
+author: mvo
+date: 2018-04-09 10:00
+tags: cockpit
+slug: making-a-cockpit-application
+category: tutorial
+comments: 'true'
+---
+
+Since version 152, Cockpit can discover and install packages that add
+pages to Cockpit.  We call them "Applications" and as of now, only two
+of them exist in regular distributions: Fleet Commander and Cockpit's
```

Comment:
I'd be more specific here: Perhaps s/regular distributions/Fedora 27 and 28/?

Reply:
Fixed.  I used "as of now, only two
of them exist: We haveFleet Commander and Cockpit's own Diagnostic
Reports in Fedora."  Otherwise the reader might get the impression that Fedora has only two, but other distributions might have more.